ckb_syscalls.h

```diff
@@ -377,6 +377,31 @@ int ckb_exec_cell(const uint8_t* code_hash, uint8_t hash_type, uint32_t offset,
                  argv);
 }
 
+int ckb_spawn(size_t index, size_t source, size_t bounds, int argc,
+              const char* argv[], spawn_args* spgs) {
+  return syscall(SYS_ckb_spawn, index, source, bounds, argc, argv, spgs);
+}
+
+int ckb_spawn_cell(const uint8_t* code_hash, uint8_t hash_type, uint32_t offset,
+                   uint32_t length, int argc, const char* argv[],
+                   spawn_args* spgs) {
+  size_t index = SIZE_MAX;
+  int ret = ckb_look_for_dep_with_hash2(code_hash, hash_type, &index);
+  if (ret != CKB_SUCCESS) {
+    return ret;
+  }
+  size_t bounds = ((size_t)offset << 32) | length;
+  return ckb_spawn(index, CKB_SOURCE_CELL_DEP, bounds, argc, argv, spgs);
```

Comment:
Do some checking on `spawn_args_t.memory_limit` and `spawn_args_t.content_length`.